src/components/Points/PointImage.jsx

```diff
@@ -15,7 +15,7 @@ function PointImage({ data, sx }) {
         return false;
       }
       if (url) {
-        return /\.(jpg|jpeg|png)$/.test(url.pathname);
+        return /\.(jpg|jpeg|png|webp|bmp|gif|ico|svg)$/.test(url.pathname);
```

Comment:
According to [this](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types) we should avoid bmp/ico formats in web content. So I'd prefer to remove those:

```suggestion
        return /\.(jpg|jpeg|png|webp|gif|svg)$/.test(url.pathname);
```

Reply:
I've accepted your suggested change; but don't fully see how general guidelines for what web devs should aim for when doing web development fully relates here.

What image formats you should _prefer_ in web dev is very different from what image preview formats make sense for a dashboard utility that allows people to preview data in their datasets/collections. If someone happens to have `.ico` icon url added as payload, why not show it?